src/data/analytics.json

```diff
@@ -98,8 +102,7 @@
           "404": {
             "$ref": "#/components/responses/IndexNotFound"
           }
-        },
-        "tags": ["analytics"]
```

Comment:
Is it expected that we lose the tags? We don't leverage them currently afaik, but it could be useful metadata 👀

Reply:
Ah good point, the tags are only there for documentation purposes but those custom methods are not part of the API reference, so not tagged